apps/designer/app/designer/features/props-panel/props-panel.tsx

```diff
@@ -200,8 +210,14 @@ export const PropsPanel = ({
       <PlusIcon />
     </Button>
   );
+
   return (
-    <>
+    <Box
+      css={{
+        position:
+          "relative" /* Hack - value picker popover positioning depends on it */,
```

Comment:
now we gotta fix it once for both cases, I think we should have that container in a location for both, seems like the closest shared folder is designer/shared/inspector/pane.tsx